profiler/CMakeLists.txt

```diff
@@ -87,6 +87,36 @@ if(THEROCK_ENABLE_ROCPROFV3)
   therock_cmake_subproject_provide_package(rocprofiler-sdk rocprofiler-sdk-roctx lib/cmake/rocprofiler-sdk-roctx)
   therock_cmake_subproject_activate(rocprofiler-sdk)
 
+  ##############################################################################
+  # rocprofiler-compute
```

Comment:
I assume this is not yet available on Windows? If so, we could add an entry to the docs in https://github.com/ROCm/TheRock/blob/main/docs/development/windows_support.md#supported-subprojects

Source line to put this next to `rocprofiler-sdk` which is also unsupported on Windows: https://github.com/ROCm/TheRock/blob/5afb2d02a5cb18f39816fbc36ba3c619a9c5c7d7/docs/development/windows_support.md?plain=1#L46